subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -62,7 +62,7 @@ static struct {
 		};
 		uint8_t all;
 	};
-} context;
+} wifi_shell_ctx;
```

Comment:
For the second commit, I am not really sure how this improves clarity. The original code looks fine by me.

Reply:
Then I will revoke this change.